### ISSUE TITLE
[tensorflow-serving] mnist-export model not working

### ISSUE THREAD

**Issue by thomasliddledba** [helm/charts#5032]
**Version of Helm and Kubernetes**:
Helm Version
 - Client 2.7.0
 - Server 2.70

**Which chart**: 
[tensorflow-serving]

**What happened**:
Performing the following command does not extract the model
```
curl -O https://github.com/cheyang/charts/raw/features/tensorflow_serving/stable/tensorflow-serving/models/mnist-export.tar.gz
```
```
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**What you expected to happen**:
Model sample was extracted out

**How to reproduce it** (as minimally and precisely as possible):
Perform the command:
```
curl -O https://github.com/cheyang/charts/raw/features/tensorflow_serving/stable/tensorflow-serving/models/mnist-export.tar.gz
```

**Comment by thomasliddledba** [comment]
CC @cheyang

**Comment by cheyang** [comment]
Please download by executing  `curl -O https://raw.githubusercontent.com/kubernetes/charts/master/stable/tensorflow-serving/models/mnist-export.tar.gz`. Sorry, I will update the README.md soon.

**Comment by cheyang** [comment]
Will fix #5033

**Comment by thomasliddledba** [comment]
@cheyang - this worked thanks.

**Comment by cheyang** [comment]
The fix #5033  is already merged. @mattfarina  Can you help to close it? thanks.

**Comment by cheyang** [comment]
Fixes #5033
